src/types/prefix_str.rs

```diff
@@ -136,91 +174,99 @@ prefix_str_type!(
 #[cfg(test)]
 mod tests {
     use crate::types::{U16PrefixStr, U16PrefixStrMut, U8PrefixStr, U8PrefixStrMut};
+    use std::str::Utf8Error;
+
+    type Result = std::result::Result<(), Utf8Error>;
 
     #[test]
-    fn test_new() {
+    fn test_new() -> Result {
         // u8
         let mut data = [0u8; 4];
-        let mut prefix_str = U8PrefixStrMut::new(&mut data);
+        let mut prefix_str = U8PrefixStrMut::new(&mut data)?;
         prefix_str.copy_from_str("str");
 
         assert_eq!(prefix_str.as_str(), "str");
         assert_eq!(prefix_str.size(), data.len());
 
         // u16
         let mut data = [0u8; 5];
-        let mut prefix_str = U16PrefixStrMut::new(&mut data);
+        let mut prefix_str = U16PrefixStrMut::new(&mut data)?;
```

Comment:
The alignment of the type is `2` in this case, but the data length that the test is using is `5`. Maybe this is causing the alignment error on CI/Linux. We could try using data length `6` here so that it is a multiple of of `2`.